scraper/visualize.py

```diff
@@ -92,6 +92,12 @@ def show_products(products: list[ProductInfo], title: str) -> None:
             name_format="%website - %name - %id",
         )
     config_figure(fig, title)
+    fig.update_layout(legend=dict(
+        yanchor="bottom",
+        y=-0.20,
+        xanchor="left",
+        x=0.01
+    ))    
```

Comment:
I would prefer if this was put inside the function `config_figure`. I have just experimented with the legend position and prefer this layout for legends; `legend=dict(orientation="v")`. Although that means that if there are many products visualized then the graph would shrink vertically instead of horizontally, but the legends will not overlap the graph itself.

Reply:
@Crinibus  Updated the legend visualisation in `scraper/visualize.py`.
<img width="1728" alt="legend problem1" src="https://github.com/Crinibus/scraper/assets/2772109/847364b4-2001-4dde-94a4-e29ec03faac5">